### PR TITLE
Projectile Dampener Module is now printable

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
## About The Pull Request

Adds projectile dampener and criminal capture to security MOD techweb node and mech printer.

## Why It's Good For The Game

These mod currently can't be printed and HOS modsuit "Safeguard" is not pre-equipped with them. Considering it's wide usage for other MODsuits and talking to fikou, I have changed the previous pr idea into adding these to the exofab. Users can now choose between their regular security MOD or take the defensive one.

## Changelog
:cl: SpaceLove
add: Projectile Dampener(1000 metal, 500 bluespace) can be researched and printed like other modules.
:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
how do you have a negative 5 delta

**Comment by SpaceLoveSs13** [comment]
> how do you have a negative 5 delta

This is my first tgstation(not tg org) pr. Since this pr is tagged with feature it gave -5 delta GBP instantly from the get go. I believe that is how it works.

Regardless, I am drafting this pr for now because I have talked to fikou and he would prefer to have safeguard equipped with pepper and instead make this pr have the criminal capture printable from robotics.

**Comment by Fikou** [comment]
> how do you have a negative 5 delta

they pr'd to tgmc before

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1532,6 +1532,8 @@
 		"mod_mag_harness",
 		"mod_pathfinder",
 		"mod_holster",
+		"mod_criminal_capture",
```
Comment (Fikou) [inline code/modules/research/techweb/all_nodes.dm:1535]: criminal capture cannot be printable while it gives you three free prisoner bags

Comment (SpaceLoveSs13) [reply code/modules/research/techweb/all_nodes.dm:1535]: You want me to make a new child mod that doesn't give capture bags?

Comment (Fikou) [reply code/modules/research/techweb/all_nodes.dm]: Either that or limit the criminal capture in some way to require the modsuit to be nearby or something like that

**Comment by JohnFulpWillard** [comment]
updated the title to match the changes you made to it, otherwise it's good

**Pull request merged**